app/src/main/java/org/mozilla/fenix/settings/logins/controller/SavedLoginsStorageController.kt

```diff
@@ -56,6 +57,48 @@ open class SavedLoginsStorageController(
         }
     }
 
+    fun add(hostnameText: String, usernameText: String, passwordText: String) {
```

Comment:
I think this function should be `suspend`, and the other one shouldn't. That will simplify the `launch` here

Reply:
I made it similar to `fun save()`
Should I still make it the other way?

Reply:
let me take a look. I guess keeping it the same way as the other stuff is fine